pkg/sink/sink.go

```diff
@@ -91,6 +94,14 @@ type Response struct {
 	ErrorMessage string `json:"errorMessage,omitempty"`
 }
 
+func (r Sink) emitEvents(ctx context.Context, el *triggersv1.EventListener, err error) {
+	var afterCondition *apis.Condition
+	if el != nil {
+		afterCondition = el.Status.GetCondition(apis.ConditionReady)
```

Comment:
I don't think we should send the `apis.ConditionReady` as the message for the event. Maybe we can try to be consistent with the cloudevents PR here.

Reply:
with `el.Status.GetCondition(apis.ConditionReady)` not sending message to event instead we are getting el Status